src/sass/_permission_confirm_modal.scss

```diff
@@ -0,0 +1,60 @@
+.permission-confirm-modal {
+  .confirm-table {
+    overflow-y: auto;
+
+    .modified-item {
+      margin-bottom: $spv--small;
+    }
+
+    .modified-row {
+      border-top: $border-thin;
+    }
+
+    .removed {
+      text-decoration: line-through;
+    }
+
+    tr {
+      border: none;
+    }
+
+    thead {
+      background-color: white;
+      border-bottom: $border-thin;
+      position: sticky;
+      top: 0;
+    }
+
+    .display-by-header {
+      position: relative;
+    }
+
+    .display-by-button {
+      bottom: 0.45rem;
+      position: absolute;
+      right: 0;
+    }
+  }
+
+  .p-modal__header {
+    margin-bottom: 0;
+  }
```

Comment:
> It looks like there's a tiny little gap above the sticky header in the confirmation modal, and you can see scrolling content through it

This was the fix for the above issue picked up by @piperdeck . For some reason on safari, table items shows up in that gap where the margin is located when scrolled above the sticky table headers.

Reply:
It's not happening on chrome though, and also margin does not have background colour?

Reply:
I see, then this seems to be a safari bug. Maybe worth adding a comment and/or mentioning this to @bartaz as it could be better to add a workaround to vanilla as well?

Reply:
Yes will add the comment and chat to bartek about this :+1: